--- gtkrc.old	2017-05-15 15:17:52.430332783 +0700
+++ gtkrc	2017-05-15 15:17:50.254332713 +0700
@@ -61,7 +61,7 @@
     GtkScale::trough-side-details              = 1
 
     GtkScrollbar::min-slider-length            = 42
-    GtkScrollbar::slider-width                 = 14
+    GtkScrollbar::slider-width                 = 7
     GtkScrollbar::activate-slider              = 1
     GtkScrollbar::trough-border                = 0
     GtkScrollbar::has-backward-stepper         = 1
